src/app/components/core/support/support.component.ts

```diff
@@ -55,5 +56,17 @@ export class SupportComponent implements OnInit {
           }
         });
   }
+
+  public async downloadBundle(bundle): Promise<void> {
+    const blob = await this.supportBundleService.downloadSupportBundle(bundle);
+    const url = window.URL.createObjectURL(blob);
+    // create a custom anchor tag
+    const a = document.createElement('a');
+    document.body.appendChild(a);
+    a.href = url;
+    a.download = bundle;
+    a.click();
```

Comment:
put `document.body.appendChild(a);` just above `a.click();` after assigining href and download attr/prop values